src/dotcom/theme.js

```diff
@@ -0,0 +1,550 @@
+      "editorCursor.foreground"           : themes({ light: scale.blue[7], dark: scale.blue[2], dimmed: scale.blue[2] }),
+
+      "editor.findMatchBackground"          : themes({ light: scale.yellow[4], dark: "#ffd33d44", dimmed: "#ffd33d44" }),
+      "editor.findMatchHighlightBackground" : themes({ light: "#ffdf5d66", dark: "#ffd33d22", dimmed: "#ffd33d22" }),
+      "editor.inactiveSelectionBackground"  : themes({ light: "#0366d611", dark: "#3392FF22", dimmed: "#3392FF22" }),
+      "editor.selectionBackground"          : themes({ light: "#0366d625", dark: "#3392FF44", dimmed: "#3392FF44" }),
+      "editor.selectionHighlightBackground" : themes({ light: "#34d05840", dark: "#17E5E633", dimmed: "#17E5E633" }),
+      "editor.selectionHighlightBorder"     : themes({ light: "#34d05800", dark: "#17E5E600", dimmed: "#17E5E600" }),
+      "editor.wordHighlightBackground"      : themes({ light: "#34d05800", dark: "#17E5E600", dimmed: "#17E5E600" }),
+      "editor.wordHighlightStrongBackground": themes({ light: "#34d05800", dark: "#17E5E600", dimmed: "#17E5E600" }),
+      "editor.wordHighlightBorder"          : themes({ light: "#24943e99", dark: "#17E5E699", dimmed: "#17E5E699" }),
+      "editor.wordHighlightStrongBorder"    : themes({ light: "#24943e50", dark: "#17E5E666", dimmed: "#17E5E666" }),
+      "editorBracketMatch.background"       : themes({ light: "#34d05840", dark: "#17E5E650", dimmed: "#17E5E650" }),
+      "editorBracketMatch.border"           : themes({ light: "#34d05800", dark: "#17E5E600", dimmed: "#17E5E600" }),
+
+      "editorGutter.modifiedBackground": color.diff.change.border,
+      "editorGutter.addedBackground"   : color.diff.addition.border,
+      "editorGutter.deletedBackground" : color.diff.deletion.border,
+
+      "diffEditor.insertedTextBackground": chroma(color.diff.addition.bg).hex(),
+      "diffEditor.removedTextBackground":  chroma(color.diff.deletion.bg).hex(),
+
+      "scrollbar.shadow"                : themes({ light: "#6a737d33", dark: "#0008", dimmed: "#0008" }),
+      "scrollbarSlider.background"      : themes({ light: "#959da533", dark: "#484F5833", dimmed: "#484F5833" }),
+      "scrollbarSlider.hoverBackground" : themes({ light: "#959da544", dark: "#484F5844", dimmed: "#484F5844" }),
+      "scrollbarSlider.activeBackground": themes({ light: "#959da588", dark: "#484F5888", dimmed: "#484F5888" }),
+      "editorOverviewRuler.border"      : themes({ light: scale.white, dark: scale.black, dimmed: scale.black }),
+
+      "panel.background"             : color.bg.canvasInset,
+      "panel.border"                 : color.border.primary,
+      "panelTitle.activeBorder"      : "#f9826c",
+      "panelTitle.activeForeground"  : color.text.primary,
+      "panelTitle.inactiveForeground": color.text.tertiary,
+      "panelInput.border"            : color.border.primary,
+
+      "terminal.foreground": color.text.secondary,
```

Comment:
Once merged the [color-modes](https://github.com/primer/primitives/tree/mkt/color-modes) branch in the [primer/primitives](https://github.com/primer/primitives) module will provide ansi colors for the terminal, we should add these to this theme
``` js
'terminal.ansiBlack': color.ansi.black,
'terminal.ansiRed': color.ansi.red,
'terminal.ansiGreen': color.ansi.green,
'terminal.ansiYellow': color.ansi.yellow,
'terminal.ansiBlue': color.ansi.blue,
'terminal.ansiMagenta': color.ansi.magenta,
'terminal.ansiCyan': color.ansi.cyan,
'terminal.ansiWhite': color.ansi.white,
'terminal.ansiBrightBlack': color.ansi.blackBright,
'terminal.ansiBrightRed': color.ansi.redBright,
'terminal.ansiBrightGreen': color.ansi.greenBright,
'terminal.ansiBrightYellow': color.ansi.yellowBright,
'terminal.ansiBrightBlue': color.ansi.blackBright,
'terminal.ansiBrightMagenta': color.ansi.magentaBright,
'terminal.ansiBrightCyan': color.ansi.cyanBright,
'terminal.ansiBrightWhite': color.ansi.whiteBright,
```